test/Test_ZProc.ml

```diff
@@ -1,6 +1,7 @@
 open Base
 open LoopInvGen
 open LoopInvGen.ZProc
+open Sexp
```

Comment:
The order in which we open modules matters. It should be: libraries, then our modules -- so that in case of a name clash, our names are not shadowed by library names.

Change the order to: `Base`, `Sexp`, `LoopInvGen`, `LoopInvGen.ZProc` (or simply `ZProc`).